lib/js-yaml/dumper.js

```diff
@@ -296,7 +287,7 @@ function chooseScalarStyle(string, singleLineOnly, indentPerLevel, lineWidth, te
     // in case the end is missing a \n
     hasFoldableLine = hasFoldableLine || (shouldTrackWidth &&
       (i - previousLineBreak - 1 > lineWidth &&
-       string[previousLineBreak + 1] !== ' '));
+      string[previousLineBreak + 1] !== ' '));
```

Comment:
i think it doesn't worth to change indent here if that's not related to your issue.

Reply:
That was eslint --fix (`make fix`) in action

Reply:
=> don't use `eslint --fix` :)

There is local `make lint` with appropriate config, and it does not report error on this line.